openshift/openshift.go

```diff
@@ -226,7 +227,7 @@ func (s *openshiftImageSource) GetBlob(info types.BlobInfo) (io.ReadCloser, int6
 	return s.docker.GetBlob(info)
 }
 
-func (s *openshiftImageSource) GetSignatures() ([][]byte, error) {
+func (s *openshiftImageSource) GetSignatures(ctx context.Context) ([][]byte, error) {
```

Comment:
@mfojtik added in https://github.com/mtrmac/image/tree/add-context-openshift , can you add that into this PR?

(Disclaimer: I’m still on Go 1.6, so this is untested. I’ll have to update ~immediately, I know.)